app/views/users/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div>
+  <div>
+    <% if @user.errors.any? %>
+       <div id="error_explanation">
+         <h2><%= pluralize(@user.errors.count, "error") %> prohibited this user from being registered:</h2>
+
+         <ul>
+         <% @user.errors.full_messages.each do |msg| %>
+           <li><%= msg %></li>
+         <% end %>
+         </ul>
+       </div>
+     <% end %>
+
+    <%= form_for(@user) do |f| %>
```

Comment:
!: You are missing the plan HTML form [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) 
2: form tag form is missing [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png)